models/articles.models.js

```diff
@@ -3,7 +3,8 @@ const db = require("../db/connection");
 exports.selectArticlesByID = (article_id) => {
   return db
     .query(
-      " select u.name as author, ar.title, ar.article_id, ar.body, ar.topic, ar.created_at, ar.votes from articles ar join users u on ar.author=u.username where ar.article_id=$1",
+      `select u.name as author, ar.title, ar.article_id, ar.body, ar.topic, ar.created_at, ar.votes,count(c.*) ::INT as comment_count from articles ar join users u on ar.author=u.username join comments c on ar.article_id=c.article_id where ar.article_id=$1 GROUP by u.name, ar.title, ar.article_id, ar.body, ar.topic, ar.created_at, ar.votes
```

Comment:
The query is fine but is very difficult to read, you should use CAPITAL words for everything that is PSQL please